web/components/bet/bet-panel.tsx

```diff
@@ -229,7 +230,7 @@ export function BuyPanel(props: {
   }
 
   async function submitBet() {
-    if (!user || !betAmount) return
+    if (!user || !betAmount || !outcome) return
```

Comment:
What case is this covering? Shouldn't the bet placement UI be disabled when they haven't picked an outcome?

Reply:
Oh, I think the submit button is disabled so it wouldn't happen, but the type of outcome is `string | undefined`, so I added this case to appease the types (plus it's a good idea anyway).

Reply:
For just appeasing the types I suggest using the Typescript `!` non-null assertion operator. Whether you think it's good style to handle it here I leave up to your judgement. I am guessing I would not put it, because I think it's confusing to readers for the code to defensively handle cases that are logically impossible.